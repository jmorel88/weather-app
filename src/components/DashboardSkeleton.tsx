export default function DashboardSkeleton() {
  return (
    <div
      role="status"
      className="p-8 bg-black-200 rounded-lg text-white-900 text-center md:text-left"
    >
      <div className="animate-pulse h-2.5 bg-gray-500 rounded-full w-48 mb-4"></div>
      <div className="animate-pulse h-2 bg-gray-500 rounded-full w-full max-w-md mb-2.5"></div>
      <div className="animate-pulse h-2 bg-gray-500 rounded-full w-full max-w-md mb-2.5"></div>
      <div className="animate-pulse h-2 bg-gray-500 rounded-full w-full max-w-lg mb-2.5"></div>
      <div className="animate-pulse h-2 bg-gray-500 rounded-full w-full max-w-xl mb-2.5"></div>
      <div className="animate-pulse h-2 bg-gray-500 rounded-full w-full max-w-lg"></div>
      <span className="sr-only">Loading...</span>
    </div>
  );
}
