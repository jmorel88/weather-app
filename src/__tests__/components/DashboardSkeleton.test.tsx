import { describe, test } from "vitest";
import { render } from "@testing-library/react";
import DashboardSkeleton from "../../components/DashboardSkeleton";

describe("Summary", () => {
  test("renders correctly", ({ expect }) => {
    const { container } = render(<DashboardSkeleton />);
    expect(container).toMatchSnapshot();
  });
});
