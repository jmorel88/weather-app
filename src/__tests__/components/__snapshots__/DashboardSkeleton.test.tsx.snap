// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Summary > renders correctly 1`] = `
<div>
  <div
    class="p-8 bg-black-200 rounded-lg text-white-900 text-center md:text-left"
    role="status"
  >
    <div
      class="animate-pulse h-2.5 bg-gray-500 rounded-full w-48 mb-4"
    />
    <div
      class="animate-pulse h-2 bg-gray-500 rounded-full w-full max-w-md mb-2.5"
    />
    <div
      class="animate-pulse h-2 bg-gray-500 rounded-full w-full max-w-md mb-2.5"
    />
    <div
      class="animate-pulse h-2 bg-gray-500 rounded-full w-full max-w-lg mb-2.5"
    />
    <div
      class="animate-pulse h-2 bg-gray-500 rounded-full w-full max-w-xl mb-2.5"
    />
    <div
      class="animate-pulse h-2 bg-gray-500 rounded-full w-full max-w-lg"
    />
    <span
      class="sr-only"
    >
      Loading...
    </span>
  </div>
</div>
`;
